### PR TITLE
Defly.io Presence Creation

### PR DESCRIPTION
**Pull request by Zt-freak**
---
Name: Defly.io Presence Creation
About: Pull request for adding Defly.io.
---

**Screenshots**

image
image

### CODE REVIEW

File: Defly.io/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    "name": "Apodemus",
+    "id": "223817321695674371"
+  },
+  "url": "defly.io",
+  "description": {
+    "en": "Fly your helicopter and shoot at enemies. Build walls to protect yourself and conquer as much territory as possible."
+  },
+  "service": "Defly.io",
+  "logo": "https://defly.io/favicon/android-icon-192x192.png",
+  "thumbnail": "https://defly.io/favicon/android-icon-192x192.png",
```
Comment (Bas950) [inline Defly.io/dist/metadata.json]: upload to imgur

File: Defly.io/dist/metadata.json [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  "color": "#3D5DFF",
+  "tags": [],
+  "category": "games"
```
Comment (Bas950) [inline Defly.io/dist/metadata.json]: you need atleast 1 tag

File: Defly.io/dist/presence.js [hunk 3]
```diff
@@ -0,0 +1,63 @@
+var __awaiter = (this && this.__awaiter) || function (thisArg, _arguments, P, generator) {
+    return new (P || (P = Promise))(function (resolve, reject) {
+        function fulfilled(value) { try { step(generator.next(value)); } catch (e) { reject(e); } }
+        function rejected(value) { try { step(generator["throw"](value)); } catch (e) { reject(e); } }
+        function step(result) { result.done ? resolve(result.value) : new P(function (resolve) { resolve(result.value); }).then(fulfilled, rejected); }
+        step((generator = generator.apply(thisArg, _arguments || [])).next());
+    });
```
Comment (Bas950) [inline Defly.io/dist/presence.js]: The awaiter can be removed see <https://docs.premid.app/en/dev/presence#getting-started-1>

File: Defly.io/dist/metadata.json (hunk 1 above)
Comment (Zt-freak) [reply Defly.io/dist/metadata.json]: new file source is: https://i.imgur.com/vTAH4DD.png

File: Defly.io/dist/metadata.json (hunk 2 above)
Comment (Zt-freak) [reply Defly.io/dist/metadata.json]: tags are now: `["game", "defly", "multiplayer", "defly.io", "browser game"]`

File: Defly.io/dist/presence.js (hunk 3 above)
Comment (Zt-freak) [reply Defly.io/dist/presence.js]: awaiter removed

File: Defly.io/dist/metadata.json [hunk 4]
```diff
@@ -8,9 +8,9 @@
     "en": "Fly your helicopter and shoot at enemies. Build walls to protect yourself and conquer as much territory as possible."
   },
   "service": "Defly.io",
-  "logo": "https://defly.io/favicon/android-icon-192x192.png",
-  "thumbnail": "https://defly.io/favicon/android-icon-192x192.png",
+  "logo": "https://i.imgur.com/vTAH4DD.png",
+  "thumbnail": "https://i.imgur.com/vTAH4DD.png",
```
Comment (Bas950) [inline Defly.io/dist/metadata.json]: thumbnail should be an image of the site itself

Comment (Zt-freak) [reply Defly.io/dist/metadata.json]: gotcha https://i.imgur.com/BwWCTIf.png

**Pull request merged**